src/modules/I18n/i18n.js

```diff
@@ -4,8 +4,9 @@ module.exports = {
   keySeparator: false, // to be able to use `.` in the key names
   logBuild: false,
   pages: {
-    '*': ['common', 'header', 'footer', 'follow-us'],
+    '*': ['common', 'header', 'footer', 'follow-us', 'case-studies'],
```

Comment:
I first implemented `breadcrumb` with frontmatter values but in this case this does not seem to me to be a necessary data since it can be generated automatically (but it can still be done when necessary with `[...paths]` updates).

I agree with you about using a dedicated layout adding the `src/pages/case-studies/[[...paths]].tsx` file.